.github/workflows/build.yml

```
@@ -78,13 +78,39 @@ jobs:
       - name: Gradle build
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: --refresh-dependencies --stacktrace --scan clean build
+          arguments: --refresh-dependencies --stacktrace --scan clean build -x spotlessCheck
       - name: SonarCloud analysis
         if: ${{ matrix.os == 'ubuntu' }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: "61ab2579215aa8a0024a2f9368fc1298fdecfd18"
         run: ./gradlew jacocoTestReport sonarqube --stacktrace -i
+      - name: Test report
+        uses: mikepenz/action-junit-report@v2
```

Current version seems to be [v3.0.0](https://github.com/mikepenz/action-junit-report/releases/tag/v3.0.0):

```suggestion
        uses: mikepenz/action-junit-report@v3
```